src/Packager.php

```diff
@@ -95,16 +99,53 @@ class Packager
      */
     private $createZipArchive;
 
+    /**
+     * Metadata description
+     *
+     * @var string
+     */
+    private $metadataDescription;
+
+    /**
+     * Metadata entry identifier
+     *
+     * @var string
+     */
+    private $entryIdentifier;
+
+    /**
+     * Metadata catalog value
+     *
+     * @var string
+     */
+    private $catalogValue;
+
+    /**
+     * Metadata lifeCycle version
+     *
+     * @var string
+     */
+    private $lifeCycleVersion;
+
+    /**
+     * Metadata classification
+     *
+     * @var string
+     */
+    private $classification;
+
     /**
      * Packager constructor.
      *
      * @param array $config
+     * @param array|null $metadataConfig
      *
      * @throws Throwable
      */
-    public function __construct(array $config)
+    public function __construct(array $config, array $metadataConfig = null)
```

Comment:
You should use null type hint if you want to use null as default value for second argument.